packages/kit/src/node/index.js

```diff
@@ -64,11 +65,21 @@ export async function getRequest(base, req) {
 		delete headers[':scheme'];
 	}
 
-	return new Request(base + req.url, {
+	const request = new Request(base + req.url, {
 		method: req.method,
 		headers,
 		body: await get_raw_body(req) // TODO stream rather than buffer
 	});
+
+	request.formData = async () => {
+		return new NodeFetchRequest(request.url, {
+			method: request.method,
+			headers: request.headers,
+			body: Buffer.from(await request.arrayBuffer())
```

Comment:
you should rather do something like `body: stream.Readable.from(request.body)` since undici uses web streams and node-fetch kind of expect a node stream

you also have https://nodejs.org/dist/latest-v18.x/docs/api/stream.html#streamreadablefromwebreadablestream-options